security.tf

```diff
@@ -121,6 +121,17 @@ resource "aws_security_group_rule" "cluster_ingress" {
   security_group_id        = local.cluster_security_group_id
 }
 
+resource "aws_security_group_rule" "private_cluster_ingress" {
+  count             = local.cluster_endpoint_private_access_cidrs != null ? length(local.cluster_endpoint_private_access_cidrs) > 0 ? 1 : 0 : 0
+  type              = "ingress"
+  description       = "Allow tcp port 443 ingress to EKS cluster API from cluster endpoint private access cidrs"
```

Comment:
for description:  "Allow additional HTTPS/443 ingress to private EKS cluster API server endpoint per var.cluster_endpoint_private_access_cidrs"

Reply:
applied